src/cs/LionWeb.Integration.WebSocket.Tests/Server/AnnotationServerTests.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System.Diagnostics;
+using LionWeb.Core.M1.Event;
+using LionWeb.Integration.Languages.Generated.V2023_1.TestLanguage.M2;
+using LionWeb.Integration.WebSocket.Server;
+using LionWeb.Protocol.Delta.Repository;
+
+namespace LionWeb.Integration.WebSocket.Tests.Server;
+
+[TestFixture(ClientProcesses.CSharp)]
+public class AnnotationServerTests(params ClientProcesses[] clientProcesses) : WebSocketServerTestBase(clientProcesses)
+{
+    /// <summary>
+    /// Adds an annotation to the specified server partition.
+    /// </summary>
+    [Test]
+    public void AddAnnotation()
+    {
+        _webSocketServer = new WebSocketServer(_lionWebVersion) { Languages = _languages };
+        _webSocketServer.StartServer(IpAddress, Port);
+
+        var serverPartition = new LinkTestConcept("a");
+        Debug.WriteLine($"Server partition: {serverPartition.PrintIdentity()}");
```

Comment:
Is this useful? If yes ok, if it's just a leftover from copy/paste please remove.

Reply:
(same in all other tests)